### PR TITLE
added routing to every view 

### PR DESCRIPTION
**Pull request by dannyshwan**

### CODE REVIEW

File: src/Router/router.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import React from 'react';
```
Comment (bmalghan) [inline src/Router/router.js:1]: Is there a reason this is a .js file and not .ts?

Comment (dannyshwan) [reply src/Router/router.js:1]: not really

Comment (dannyshwan) [reply src/Router/router.js:1]: it was just generated that way

**Review by Keitech — APPROVED** [review]
lgtm for now : D

**Pull request merged**